packages/matter-node.js/src/storage/StorageBackendDisk.ts

```diff
@@ -9,19 +9,24 @@ import { LocalStorage } from "node-localstorage";
 
 export class StorageBackendDisk extends SyncStorage {
     private readonly localStorage;
+    protected _initialized = false;
```

Comment:
Is "_" convention for protected variables?

Reply:
*cough* ... suuuure ;-)

changed